package.json

```diff
@@ -1,78 +1,81 @@
-{
-    "name": "iamus-metaverse-server",
-    "version": "2.4.10",
```

Comment:
The name should change to "vircadia-metaverse", the author should be upgraded to "contributors", and the URL should be changed to "https://git.hub/vircadia/vircadia-metaverse" but I will include all those changes in a separate PR.